docs/admin/install.rst

```diff
@@ -650,6 +650,28 @@ After enabling it, you should set :setting:`ENABLE_HTTPS` in the settings:
    :setting:`ENABLE_HTTPS`,
    :ref:`production-site`
 
+
+Set properly SECURE_HSTS_SECONDS
+++++++++++++++++++++++++++++++++
+
+If your site is served over SSL, you have to consider setting a value for :setting:`SECURE_HSTS_SECONDS`
+in the settings.py to enable HTTP Strict Transport Security.
+By default its set to 0 as shown below.
+
+.. code-block:: python
+
+   SECURE_HSTS_SECONDS = 0
+
+If set to a non-zero integer value, the `SecurityMiddleware
+<https://docs.djangoproject.com/en/3.0/ref/middleware/#django.middleware.security.SecurityMiddleware>`_
```

Comment:
Using Sphinx links is better - it will make the links work for future Django releases as well. I think following should work (but not tested):

```suggestion
If set to a non-zero integer value, the :class:`django:~django.middleware.security.SecurityMiddleware`
```

Reply:
It works after removing the tilde ~. I have updated the PR.